logging/google/cloud/logging/logger.py

```diff
@@ -379,13 +410,17 @@ def log_text(self, text, labels=None, insert_id=None, severity=None,
 
         :type timestamp: :class:`datetime.datetime`
         :param timestamp: (optional) timestamp of event being logged.
+
+        :type resource :class:``google.cloud.logging.resource`
```

Comment:
This

- is missing a colon after `resource`
- has an extra backtick before `google` (two instead of one)
- refers to a module not a class
- could benefit from a `~` before `google`

It seems you've got a few other references to `google.cloud.logging.resource` instead of `google.cloud.logging.resource.Resource`. Can you do another pass and find-replace them?